README.md

```diff
@@ -42,9 +42,11 @@ Where a field definition object takes the following form:
   placeholder: String data (optional)
   metered: boolean (optional),
   optional: boolean (optional),
+  options: array of strings for each option when "type" is choiceGroup, checkboxGroup, or a React component that takes "options" (like a ReactSelect component),
```

Comment:
this is a @mmmavis problem 😉 I'm trying to understand what `options` is for here. Is each `option` string here the label text we show on the UI?